addons/account/tests/test_account_account.py

```diff
@@ -688,22 +688,64 @@ def test_account_code_mapping(self):
         with Form(account) as account_form:
             # Test that the code mapping gives correct values once the form has been opened (which should call search)
             self.assertRecordValues(account.code_mapping_ids, [
+                {'company_id': company_3.id, 'code': False},
```

Comment:
Is this on purpose/knowingly?
The best would be to have the mappings sorted by company sequence,name (default `_order`)

Reply:
Thanks! I've added an explicit ordering to the code_mapping_ids in the `_search` override :smile: 
The weird order of the mapping lines in this test is actually because `'Test Company 3' < 'company_1_data'` in Python.
It must have been working before by coincidence :sweat_smile: 
I've renamed the company created by the test so that the companies are in order company_1_data, company_2, company_3. :smile: